CONTRIBUTING.md

```diff
@@ -126,6 +127,7 @@ Such plugins are usually configured via vim globals, but often have no configura
 | **name**                     | The name of the Vim plugin.                                                                                                                                                                                                                        | Yes      | N/A                                   |
 | **url**                      | The URL of the plugin repository.                                                                                                                                                                                                                  | Yes      | `package` parameter's `meta.homepage` |
 | **colorscheme**              | The name of the colorscheme.                                                                                                                                                                                                                       | No       | `name` parameter                      |
+| **dependencies**             | Which dependencies to enable by default with this plugin. (List of strings)                                                                                                                                                                          | No       | `"extraConfigLuaPre"` if `isColorscheme` then `extraConfigLuaPre`, otherwise `"extraConfigLua"` |
```

Comment:
```suggestion
| **dependencies**             | A list of dependencies to enable by default with this plugin. (List of strings)                                                                                                                                                                          | No       | `[ ]` |
```

(whitespace is probably wrong)

(Again, should we have a `` [`dependencies`] `` link?)